### PR TITLE
chore: update versions in .pre-commit-config.yaml

### PR DESCRIPTION
**Pull request by eduardo-rodrigues**
Just autoupdated :).

### CODE REVIEW

**Comment by jonas-eschle** [comment]
Thanks for that, I would suggest to add it to the pre-commit CI and do the autoupdate hook, fine @andrzejnovak ?

**Comment by eduardo-rodrigues** [comment]
Strange, there is a dependabot file, I now see - https://github.com/scikit-hep/mplhep_data/blob/master/.github/dependabot.yml. Is it not running?

**Comment by eduardo-rodrigues** [comment]
Fixed the CI while at it.

**Comment by henryiii** [comment]
Dependabot only runs on GHA and/or pip, doesn’t support pre-commit.

File: setup.cfg [hunk 1]
```diff
@@ -21,9 +21,6 @@ classifiers =
     Programming Language :: Python
     Programming Language :: Python :: 3
     Programming Language :: Python :: 3 :: Only
-    Programming Language :: Python :: 3.7
-    Programming Language :: Python :: 3.8
-    Programming Language :: Python :: 3.9
```
Comment (henryiii) [inline setup.cfg:26]: Normally I don’t like these being removed, but since this is not user facing and doesn’t have much Python, I think it’s okay.

Comment (henryiii) [reply setup.cfg:26]: It’s now an optional flag. See https://github.com/scikit-hep/boost-histogram/blob/5c7ae1ca9a849ebf53cad5b7441a0d33e6e66b8d/.pre-commit-config.yaml#L35

I’d usually add that, but here I think it’s okay to leave it off due to reasons above.

Comment (eduardo-rodrigues) [reply setup.cfg:26]: Ah, thanks for letting me know about the option. Indeed agreed that here is fine :-).

**Comment by eduardo-rodrigues** [comment]
Allow me to use my karma and merge since 2 people seem happy :-).

**Comment by henryiii** [comment]
I've enabled pre-commit.ci on this repo, now, too.

**Pull request merged**